phac-nml/irida-next
# Member destroy turbostream

## What does this PR do and why?
_Describe in detail what your merge request does and why._

This PR updates group/project members destroy to work via turbo_stream.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Login as any user
2. Add a member as a direct member to a subgroup where the user is also a member of an ancestor
3. Now remove this user from the subgroup
4. Verify that the member's ancestor inheritance is displayed
5. Verify in the network tab of the browser dev tools that the request/ was sent via turbo_stream
6. Verify page is not reloaded but rather the user's inherited membership is appended to the table

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

  the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.
